### DISCUSSION TITLE
How to read and generate the identification consistency figures

### DISCUSSION THREAD

**Discussion by andrewlai** [vdemichev/DiaNN#654] [General]
I'm sorry for the question, but I am unsure how to interpret the figures in the PDF report.

![image](https://user-images.githubusercontent.com/15118374/229476135-0f6c0a88-0a05-4034-be7a-4c4bf7b4332b.png)

**Comment by vdemichev** [comment]
Y-axis is the number of IDs with <= x missing values.
